code/_onclick/hud/screen_objects.dm

```diff
@@ -76,12 +76,12 @@
 	screen_loc = ui_building
 
 /obj/screen/area_creator/Click()
-	if(usr.incapacitated())
-		return 1
+	if(usr.incapacitated() || isobserver(usr))

```

Comment:
`!IsAdminGhost(usr` imo

Reply:
I suppose it won't hurt but there's an already admin verb for creating areas.